examples/demo_vni.pp

```diff
@@ -0,0 +1,23 @@
+# Manifest to demo cisco_vni provider
```

Comment:
Couple things here.

* Can you combine all vxlan related providers into one file called demo_vxlan.pp?  Currently this would include `cisco_vxlan_global` and `cisco_vni`.  We will add `cisco_vxlan_vtep` and `cisco_vxlan_vtep_vni` later.
* Add an entry in `demo_all.pp` to call `demo_vxlan.pp`